src/app/ui-helper/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -140,4 +142,12 @@ export class EntitySubrecordComponent implements OnInit, OnChanges {
     col.selectValues = col.allSelectValues.filter(v => v.value.includes(input) || v.label.includes(input));
   }
 
+  changeVisibilityOfAddButton() {
+    if (this.showButton === true) {
```

Comment:
Can be replaced with this.showButton = !this.showButton

Reply:
Done